src/core/command/slash-command.ts

```diff
@@ -203,6 +203,29 @@ export type SlashCommandOptionConfigBase = {
   required?: boolean
 }
 
+// I don't know where to put this, help
```

Comment:
Not sure where to put this helper function, of it needs to exist at all

Reply:
It can stay here since it's not used elsewhere, but remove the comment lol